src/cdk/activation.ts

```diff
@@ -34,25 +26,14 @@ export async function activate(activateArguments: { extensionContext: vscode.Ext
     activateArguments.extensionContext.subscriptions.push(view)
 
     // Indicates workspace includes a CDK app and user has expanded the Node
-    const appNodeExpanded = view.onDidExpandElement(e => {
-        if (e.element instanceof AppNode && !e.element.expandMetricRecorded) {
-            e.element.expandMetricRecorded = true
-            recordCdkAppExpanded()
-        }
-    })
-    activateArguments.extensionContext.subscriptions.push(appNodeExpanded)
-
-    // Indicates CDK explorer view was toggled through configuration setting
-    const explorerEnabledToggled = vscode.workspace.onDidChangeConfiguration(e => {
-        if (e.affectsConfiguration(EXPLORER_ENABLED_CONFIG_KEY)) {
-            if (vscode.workspace.getConfiguration().get(EXPLORER_ENABLED_CONFIG_KEY)) {
-                recordCdkExplorerEnabled()
```

Comment:
Are these metrics still valuable and if so, can we record them some other way?

Alternately, if they're not/not possible to record otherwise, should we remove them from the telemetry definitions?

Reply:
The metrics `recordCdkExplorerEnabled` and `recordCdkExplorerDisabled` are not valuable because:

- showing/hiding a panel is trivial (unlike before where customer had to hunt for the setting)
- what matters is `recordCdkAppExpanded`, which shows actual usage
- VSCode API makes it impossible to handle _explicit_ user "hide"  event, currently

> should we remove them from the telemetry definitions

Is it potentially needed by other toolkits? I don't see any harm in leaving the old metrics names.

Reply:
We're the only toolkit with a CDK integration. Could go either way; regardless, if you don't find it valuable, would you find it valuable in a different IDE?

Reply:
I don't know. It's a high cost to calculate the needs of every IDE whenever I make a telemetry change in 1 IDE.

If I remove the metric now, but some other toolkit really wants it, that's a lot of waste effort--and it greatly increases the potential for metrics names with the same semantics but different ids.

Reply:
These currently sit in vscode definitions, so you can remove them or change them any way you want (see https://github.com/aws/aws-toolkit-common/blob/master/telemetry/definitions/vscodeDefinitions.json#L32)

Reply:
but if they would be moved to common, we would want to use the same names...